canister/src/state.rs

```diff
@@ -400,7 +400,7 @@ mod test {
 
         // Ingest the next block. This time, the performance counter is set so that
         // the ingestion is time-sliced.
-        crate::runtime::set_performance_counter_step(1_000_000_000);
+        crate::runtime::set_performance_counter_step(100_000_000);
```

Comment:
What is the reason we decreased the counter step here 10 times, why everywhere else it is halved?